src/Sylius/Bundle/OrderBundle/Resources/config/services.xml

```diff
@@ -61,5 +63,27 @@
         </service>
 
         <service id="sylius.order_processing.order_processor" class="Sylius\Component\Order\Processor\CompositeOrderProcessor" />
+
+        <service id="sylius.listener.session_cart" class="%sylius.listener.session_cart.class%">
```

Comment:
I'm wondering whether we should remove `.class` parameters in this PR or leave it for another one, it's up to you.